scripting/include/cbasenpc/entityfactory.inc

```diff
@@ -56,6 +56,24 @@ typeset CEntityFactoryOnRemoveCallback
 	function void (CBaseEntity entity);
 }
 
+typeset CEntityFactoryNextBot
+{
+	function INextBot (Address link);
+};
+
+/**
+ * Function to be used with CEntityFactory.AttachNextBot
+ *
+ * @param link          Address of the linked CBaseEntity
+ * @return              The created ToolsNextBotPlayer
+ */
+native INextBot _ToolsNextBotPlayer_Factory(Address link);
+
+stock INextBot ToolsNextBotPlayer_Factory(Address link)

```

Comment:
A stock of the same name is already defined in nextbot.inc, leading to conflicts.
https://github.com/TF2-DMB/CBaseNPC/blob/28fe10017fcffe431a7941c0e701c0268b740622/scripting/include/cbasenpc/nextbot.inc#L414-L417